src/Turbo/Twig/StreamExtension.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Kévin Dunglas <kevin@dunglas.fr>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Turbo\Twig;
+
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+/**
+ * Twig helpers to generated the Turbo Stream elements.
+ *
+ * @author Kévin Dunglas <kevin@dunglas.fr>
+ *
+ * @experimental
+ */
+final class StreamExtension extends AbstractExtension
+{
+    private const ACTIONS = [
+        'append' => true,
+        'prepend' => true,
+        'replace' => true,
+        'update' => true,
+        'remove' => true,
+    ];
+
+    private $mercureHub;
+
+    public function __construct(?string $mercureHub = null)
+    {
+        $this->mercureHub = $mercureHub;
+    }
+
+    public function getFunctions(): iterable
+    {
+        yield new TwigFunction('turbo_stream_start', [$this, 'turboStreamStart'], ['is_safe' => ['html']]);
```

Comment:
In the same way as Frame, I don't think we need to provide helpers to write the turbo stream tags. They are quite easy to use and it would help understand the way it work IMO. It would also help people use documentation from Turbo itself.

We do want to keep the ability to stream from though, so this Twig extension is definitely important.